### PR TITLE
Standardize DB Job names

### PR DESCRIPTION
**Pull request by intlabs**
Standardize the DB creation job name to <service>-db-init for all services.

Builds on the following which should be reviewed/merged first:
 * https://github.com/att-comdev/openstack-helm/pull/137
 * https://github.com/att-comdev/openstack-helm/pull/138

---
This change is [Reviewable](https://reviewable.io/reviews/att-comdev/openstack-helm/139)

### CODE REVIEW

File: glance/templates/job-db-init.yaml [hunk 1]
```diff
@@ -3,21 +3,21 @@
...
+{{ tuple $envAll $dependencies | include "dep_check_init_cont" | indent 10 }}
         ]'
     spec:
       restartPolicy: OnFailure
       nodeSelector:
         {{ .Values.labels.node_selector_key }}: {{ .Values.labels.node_selector_value }}
       containers:
-        - name: glance-init
-          image: {{ .Values.images.init }}
+        - name: glance-db-init
```
Comment (v1k0d3n) [inline glance/templates/job-db-init.yaml]: good catch on these. good work.

File: glance/templates/job-db-init.yaml [hunk 2]
```diff
@@ -3,21 +3,21 @@
...
+          image: {{ .Values.images.db_init }}
```
Comment (v1k0d3n) [inline glance/templates/job-db-init.yaml]: again, defining the assumed vars better. nice.

File: neutron/templates/job-db-init.yaml [hunk 3]
```diff
@@ -0,0 +1,32 @@
+{{- $envAll := . }}
```
Comment (v1k0d3n) [inline neutron/templates/job-db-init.yaml]: this is in line with your other cleanup work. nice job separating these like your other recent commits.

**Comment by larryrensing** [comment]
looks good to me

**Pull request merged**